CMakeLists.txt

```diff
@@ -48,26 +48,16 @@ if(CMAKE_ENABLE_DEBUG)
     target_compile_options(JasmineGraph PRIVATE -g)
 endif()
 
-target_link_libraries(JasmineGraph /usr/local/lib/libmetis.so)
-target_link_libraries(JasmineGraph /usr/local/lib/libxerces-c-3.2.so)
 #TODO: Need to install the library separately
 target_link_libraries(JasmineGraph sqlite3)
 target_link_libraries(JasmineGraph pthread)
 target_link_libraries(JasmineGraph rdkafka)
-#target_link_libraries(JasmineGraph cppkafka)
 target_link_libraries(JasmineGraph Threads::Threads)
-include_directories("$ENV{HOME}/software/flatbuffers/include")
-include_directories("$ENV{HOME}/software/xerces/include")
-include_directories("$ENV{HOME}/software/spdlog/include")
-include_directories("$ENV{HOME}/software/jsoncpp/include")
-include_directories("$ENV{HOME}/software/cppkafka/include")
-include_directories("$ENV{HOME}/software/nlohmann_json/include")
+#include_directories("$ENV{HOME}/software/cppkafka/include")
```

Comment:
@thevindu-w Why do we comment the dependency for cpp Kafka? Isn't that required for handling streaming graphs?

Reply:
Since the cppkafka header files are already in the default include directories, No need to explicitly specify the path.
Note: When building `cppkafka` it's copying the files to the `/usr/local/` directory which is a default include directory.

Reply:
Ack